docs/app-directory/overview.md

```diff
@@ -7,18 +7,75 @@ hide_title: true
+
+
+### For the user
+
+
+#### Easier to find app info
+
+Your appD is the one place to collect all the information about apps. The more apps you have, the more you’ll appreciate the convenience of not having to chase down details about each. This is particularly important for large institutions with multiple desks.
+
+
+#### Human readable
+
+AppD has two types of users. One is the desktop agent, but the other is humans administrating and using the smart desktop at your organization. Hence, an appD contains information about apps in both machine- and human-readable forms. For example, it includes both unique identifiers for apps that are used to refer to them in code and human-friendly app names, icons, descriptions and tooltips necessary to populate a launcher menu or app catalog user interface for your users. 
+
+
+#### Apps are discoverable
+
+For large institutions, it can be difficult to keep track of all the apps (developed both in-house and by vendors), since a typical desktop could have many. Users can search appD to discover the apps they need. An app might already reside on their system or be available to them over the internet, but if they don’t have a way to search the apps available to them, they won’t be able to find it. An appD provides a way for users to discover the apps they need.
+
+An appD makes it possible to discover info about apps that reside on various domains, not just the one domain the appD itself is hosted on. In addition, you can find details about how to launch the apps in multiple, diverse environments. This is in sharp contrast to the [W3C's Web App Manifest](https://www.w3.org/TR/appmanifest/), which is hosted on the same domain as the app, is limited to web apps, and is generally used to 'install' an app that the user has already accessed.
+
+
+#### Updating apps
+
+Typically, software evolves over time. The app versions you are running today will not be the same ones you need tomorrow. Therefore, you will need to upgrade apps periodically. Very few people look forward to upgrading, but appD and web deployment can make it easier for you. To roll out a new version of your app, either update the existing entry for it in appD or add a new entry for that version (allowing users to select the version they will use).
+
+
+#### Agent-agnostic
+
+As a part of the FDC3 standard, appD isn't tied to any specific vendor. This is important, as it allows you more flexibility in that you are not tied to any specific container or desktop agent implementation. If at any point you want to switch to a different desktop agent, the process won’t be prohibitively difficult. The existing appD will work without any additional effort from you, as long as your new desktop agent is also FDC3-compliant. This is in contrast with proprietary solutions, where you would have to reconstruct the entire ecosystem from scratch. 
```

Comment:
"From scratch" might be a bit harsh - I would suggest replacing the last sentence with "Compare with proprietary solutions, where it's not guaranteed that the data structure would work with other desktop agents"

Reply:
But it is stronger than that - if you're using a proprietary format and move to a different vendor that uses a different format it's not going to work by serendipity. Hence 'not guaranteed' is too much of a mitigation in my opinion.

Would swapping out "reconstruct" for "reconfigure" help mitigate @openfin-johans ?

Reply:
But the statement is in fact true - if you use a proprietary solution to integrate everything, then try to switch provider you have to write new configs replacing the proprietary ones... this is the primary benefit of using the standard over a proprietary solution; so I'm not sure I see the issue. If its the wording, we can try other formulations, e.g.:

```suggestion
As a part of the FDC3 standard, appD isn't tied to any specific vendor. This is important, as it allows you more flexibility in that you are not tied to any specific container or desktop agent implementation. If at any point you want to switch to a different desktop agent, the process won’t be prohibitively difficult. The existing appD will work without any additional effort from you, as long as your new desktop agent is also FDC3-compliant. This is in contrast with proprietary solutions, where you would have to produce a new configuration and integration for every application. 
```

That said, the concept is already stated in the rest of the paragraph so I don't care all that much. It just seems to drive the point home a bit more clearly ;-)

Reply:
Happy with your suggestion here - let's ship it!

Reply:
Thank you sir, done!